app/tests/V2/test.py

```diff
@@ -0,0 +1,262 @@
+"""Contain test cases for the endpoints."""
```

Comment:
How comes both `app/tests/v2/test.py` and `app/tests/v1/test.py` have tests for v2 endpoints?

Reply:
The tests for version 2 should be inside the V2 folder, I have done exactly that, moved all version 2 test cases to a file test.py inside V2. I also added two more tests for testing User registration endpoint.